src/sst/elements/vanadis/decoder/vriscv64decoder.h

```diff
@@ -634,8 +646,15 @@ class VanadisRISCV64Decoder : public VanadisDecoder
                     } break;
                     case 0x1:
                     {
-                        // MULHSU
-                        // NOT SURE WHAT THIS IS?
+                        // MULHU mul und place in upper XLEN bits, unsigned 
+                        // need to check the register order
+                        output->verbose(
+                            CALL_INFO, 16, 0, "-------> MULHU %" PRIu16 " <- %" PRIu16 " + %" PRIu16 "\n", rd, rs1, rs2);
+
+                        bundle->addInstruction(
+                            new VanadisMultiplyHighInstruction<uint64_t,uint64_t,uint64_t,__uint128_t,false,false>(
+                                ins_address, hw_thr, options, rd, rs1, rs2));
```

Comment:
I don't think there are any 64bit operations with `MULH` series of instructions, they are all 32bit (I think)? I think this operation should be `uint32_t` based?